### DISCUSSION TITLE
Track progress of achievements

### DISCUSSION THREAD

**Discussion by TheMrKochan** [sgkoishi/PackTracker#6] [Ideas]
it would be useful to track the completion of achievements like "deal 75 damage with the power of the hero mage". Tell me, is it possible to implement this with an extension or is it more relevant to the HDT itself

**Comment by sgkoishi** [comment]
I've never looked into that and not sure where those data were stored.
HDT uses HearthMirror to read the game, but they choose to keep it private; I guess you may need some hacky way to obtain that information (if it really exists); just like what I did for the historical pack count.
https://github.com/sgkoishi/PackTracker/blob/7054e0dfc194b1317a996c329d9d9394791b8376/PackHistorian/PackWatcher.cs#L129-L180
